### PR TITLE
Add option to set a max y-axis value

### LINKED ISSUE #145: Allow users to fix y-axis
To enable reporters to compare scenarios they need to be able to fix the y and x axis.

### PR DESCRIPTION
**Pull request by nickcanz**
This PR adds an input value on the sidebar to set a fixed Y-axis value. This sets all the graphs to the same Y-axis value, let me know if that makes sense @Mdraugelis.

Closes https://github.com/CodeForPhilly/chime/issues/145

### CODE REVIEW

**Comment by quinn-dougherty** [comment]
@nickcanz you nailed it in the charting functions in `presentation.py`! i don't understand why add `max_y_axis` to Parameters object. Could you give me some insight into what you're gaining by doing this? FWIW, it's also what's causing the test to fail. At this pass, I think the move is to remove `max_y_axis` from Parameters object, but i'm open to being wrong

File: penn_chime/presentation.py [hunk 1]
```diff
@@ -396,7 +404,9 @@ def new_admissions_chart(
     )
 
 
-def admitted_patients_chart(alt, census: pd.DataFrame, plot_projection_days: int) -> alt.Chart:
+def admitted_patients_chart(alt, census: pd.DataFrame, plot_projection_days: int,
```
Comment (quinn-dougherty) [inline penn_chime/presentation.py]: can you please edit this to 
```
def func(
    arg1, 
    arg2, 
    arg3
) - > returntype: 
```
style?

**Comment by quinn-dougherty** [comment]
> i don't understand why add max_y_axis to Parameters object.

actually i changed my mind. you were right, let's keep all UI handled by the `display sidebar` func and `Parameters` class. please ignore that comment

**Comment by quinn-dougherty** [comment]
it shouldn't be too much trouble to make this test pass
```
penn_chime/presentation.py:211: in display_sidebar
    return Parameters(
E   TypeError: __init__() missing 1 required keyword-only argument: 'max_y_axis'
```

**Comment by quinn-dougherty** [comment]
@nickcanz  also heads up i did some manual rebasing, nothing major, be sure to fetch

File: penn_chime/presentation.py (hunk 1 above)
Comment (nickcanz) [reply penn_chime/presentation.py]: Done!

**Comment by quinn-dougherty** [comment]
@nickcanz  how did you decide to set 500 to the static value?

**Comment by nickcanz** [comment]
@quinn-dougherty excellent question! I just picked that as something that was greater than the current max Y value in the first graph, so it was totally random. The "default" is 0 so I wanted to pick a default value that didn't drastically change the first graph.

Obviously, that value doesn't make total sense for the other graphs on the page. It's easier coding-wise for all the graphs to use the same Y-max, but if it's more confusing, then I can definitely add an input for each graph separately.

**Comment by quinn-dougherty** [comment]
please add `step=10` or maybe 25 to the y_max slider. 

a potential problem i foresee is people checking the box but not scrolling further down on the sidebar to see that they can customize it. The reason i foresee this is because i just did it.

**Comment by nickcanz** [comment]
👍 done!

**Comment by quinn-dougherty** [comment]
A bigger issue is that it violates our aspirant pattern of not needing to pass the `Parameters` instance into the charting funcs, but i'm willing to let that slide in favor of shipping this now. ccing @jlubken

**Review by quinn-dougherty — APPROVED** [review]

**Pull request merged**